src/main/java/com/example/bookstore/model/Book.java

```diff
@@ -0,0 +1,28 @@
+package com.example.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Table(name = "book")
+@Data
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(nullable = false)
+    private String title;
+    @Column(nullable = false)
+    private String author;
+    @Column(nullable = false, unique = true)
+    private String isbn;
+    private BigDecimal price;
```

Comment:
could price be null? 
<img width="353" alt="image" src="https://github.com/holsoni/book-store/assets/32339836/cbd86573-c77b-46b7-bdc7-f271fe9f980e">